packages/expo-cli/src/commands/build/AndroidBuilder.js

```diff
@@ -61,7 +61,7 @@ See https://docs.expo.io/versions/latest/distribution/building-standalone-apps/#
   }
 
   async _clearCredentials() {
-    const username = this.user.username;
+    const username = this.manifest.owner || this.user.username;
```

Comment:
Looks like `IOSBuilder` has some similar code that assumes username is `this.user.username`: https://github.com/expo/expo-cli/blob/83a758a75892200b3bb517e389ea8093e16e23a2/packages/expo-cli/src/commands/build/ios/IOSBuilder.js#L54-L59

Can you check if it needs to be changed as well?